sdk/python/README.md

```diff
@@ -67,6 +67,121 @@ Class | Method | HTTP request | Description
 
 ## Documentation For Models
```

Comment:
Is there a way to turn off the generation of documentation for these?

Reply:
Yeah, let me actually follow what we do for Kubeflow Trainer to remove it.

Reply:
This looks good!